jwql/website/apps/jwql/bokeh_containers.py

```diff
@@ -282,10 +282,10 @@ def generic_telemetry_plot(times, values, name, nominal_value=None, yellow_limit
     if nominal_value is not None:
         fig.line(times, np.repeat(nominal_value, len(times)), line_dash='dashed')
 
-    fig.xaxis.formatter = DatetimeTickFormatter(hours=["%d %b %H:%M"],
-                                                days=["%d %b %H:%M"],
-                                                months=["%d %b %Y %H:%M"],
-                                                years=["%d %b %Y"],
+    fig.xaxis.formatter = DatetimeTickFormatter(hours="%d %b %H:%M",
```

Comment:
How did this function ever work if DateTimeTickFormatter was never imported?

Reply:
This function never worked and is not called.  bokeh_containers.py is NOT in good shape, but this isn't the PR to fix it.